values-schema.yaml

```diff
@@ -890,14 +1098,6 @@ properties:
             items:
               type: string
             type: array
-          logLevel:
```

Comment:
is this removed from the chart? or did it not exist there before at all?

Reply:
ah, I think that I redefined it schema.

Reply:
This definition has been enriched with `trace` and is available a bit higher under definitions property.